src/inspect_ai/model/_providers/hf.py

```diff
@@ -71,6 +71,9 @@ def collect_model_arg(name: str) -> Any | None:
         tokenizer_path = collect_model_arg("tokenizer_path")
         self.batch_size = collect_model_arg("batch_size")
         self.chat_template = collect_model_arg("chat_template")
+        self.tokenizer_call_args = collect_model_arg("tokenizer_call_args")
```

Comment:
How about just `tokenizer_args`?

Reply:
@alexandrasouly-aisi Are you okay w/ the change to `tokenizer_args`?